rakelib/typecheck.rake

```diff
@@ -2,6 +2,8 @@
 
 namespace :typecheck do
   task tapioca: :templates do
+    Rake::Task["compile:prism"].invoke
```

Comment:
This is supposed to run task dependencies and the ask only if they haven't been ran. Looked like what we want.